php-checks/src/test/resources/checks/ExecCallCheck.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+  foo($command);
+  exec($command); // Noncompliant
```

Comment:
Could you add the message as well?

Reply:
Changed in cdf2c99, although asserting the message has a low value to me when the message is always the same.